src/test/java/pl/gov/coi/pomocua/ads/myoffers/MyOffersResourceTest.java

```diff
@@ -92,6 +96,17 @@ void shouldReturnNotFoundWhenGettingOfferForOtherUser() {
         assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NOT_FOUND);
     }
 
+    @Test
+    void shouldReturnInactiveOffers() {
+        UserId accommodationOfferUserId = new UserId("accommodation offer user id");
+        testCurrentUser.setCurrentUserId(accommodationOfferUserId);
+        AccommodationOffer accOffer = postOffer(AccommodationsTestDataGenerator.sampleOffer(), "accommodations", AccommodationOffer.class);
+        deleteOffer("accommodations", AccommodationOffer.class, accOffer.id);
+        ResponseEntity<BaseOffer> response = restTemplate.getForEntity("/api/secure/my-offers/" + accOffer.id, BaseOffer.class);
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+        assertThat(response.getBody().status).isEqualTo(BaseOffer.Status.INACTIVE);
```

Comment:
Sorry for confusion, I confirmed that inactive offers should not be visible in the system at all including my offers page, get single offer and list offers.